packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -162,6 +171,19 @@ class CreateCommand extends FlutterCommand {
     return null;
   }
 
+  Future<String> _fetchSampleFromServer(String sampleId) async {
+    // Sanity check the sampleId
+    if (sampleId.contains(RegExp(r'[^-\w\.]'))) {
+      throwToolExit('Sample ID "$sampleId" contains invalid characters. Check the ID in the '
+        'documentation and try again.');
+    }
+
+    final String host = FlutterVersion().channel == 'beta'
```

Comment:
We're going to need to update things like these to say 'release' once we reach 1.0.  We could pre-factor to extract a `stableChannel` getter (returning a constant 'beta' now) to enable us to possibly detect the right stable channel once we publish to the release channel.  ... and at the very least, it gives us a single place to update when the time comes.

FYI @Hixie

Reply:
Is it called "release" or "stable"? I've heard both.